3_bidirectional_cxx_interop/cmake/modules/AddSwift.cmake

```diff
@@ -61,6 +66,8 @@ function(_swift_generate_cxx_header target header)
       -module-name "${ARG_MODULE_NAME}"
       -cxx-interoperability-mode=default
       -emit-clang-header-path ${header_path}
+      -emit-dependencies
```

Comment:
We'll need to update the working directory so that it writes the emitted deps file to the build directory instead of to the source directory or we will get a cmake warning:

```
CMake Warning:
  Expected depfile does not exist.

    /Users/ewilde/Work/swift-cmake-examples/3_bidirectional_cxx_interop/build/lib/fibonacci/SwiftFibonacci.emit-module.d
```

`WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}`

This also means that we'll need to pass the sources files as absolute paths instead of relative paths to the current source directory. Updating the `_AllSources` generator expression on line 56 like so should fix it:

`set(_AllSources $<PATH:ABSOLUTE_PATH,$<TARGET_PROPERTY:${target},SOURCES>,${CMAKE_CURRENT_SOURCE_DIR}>)`